### DISCUSSION TITLE
Mobile GitHub SSO

### DISCUSSION THREAD

**Discussion by AlbertKes** [community/community#14393] [Mobile] [answered]
After enable SSO, mobile app can't  search related repos.

**Comment by iskae** [comment, marked as answer]
Hi @AlbertKes! Thanks for your feedback 🎉 
It seems like the SSO has been enabled after logging in the app 👍  Can you please logging out and signing in once again? So sorry for the inconvenience.

**Reply by AlbertKes** [reply]
Thank you very much, it works.
